source/iostreams.tex

```diff
@@ -15058,7 +15082,9 @@
 \end{codeblock}
 
 \pnum
+\indextext{\idxhdr{inttypes.h}}%
```

Comment:
(By contrast, it is crucial that these index entries not introduce whitespace, so that the paragraph begins on the left margin.)